ftw/shop/browser/templates/edit_variations.pt

```diff
@@ -203,7 +203,7 @@
                             </table>
                         </fieldset>
                     </div>
-                    <input type="submit" name="form.button.save" value="Speichern"/>
+                    <input type="submit" name="form.button.save" value="Save" i18n:translate="label_save"/>
```

Comment:
Same here, `i18n:translate="value label_save"`, otherwise the translation adds new *content* inside the `<input />` instead of affecting the `value` attribute.